### ISSUE TITLE
setting the learning rate with callbacks

### ISSUE THREAD

**Issue by mjvakili** [spacer730/MLP-Reduction#1]
You can set the _lr_ using the following procedure:

`
tf.keras.backend.clear_session()` 

model = tf.keras.Sequential([....])                      # Insert your model here 

lr_schedule = tf.keras.callbacks.LearningRateScheduler(
    lambda epoch: 1e-6 * 10**(epoch / 20))

optimizer = tf.keras.optimizers.Adam(lr=1e-6)  # Choose your own optimizer here

model.compile(loss=...,                                     # Choose your own loss here
              optimizer=optimizer,
              metrics=["mae"])

history = model.fit(trainX, trainY, epochs=60, callbacks=[lr_schedule])

plt.loglog(history.history["lr"], history.history["loss"])

plt.xlabel("Learning rate")

plt.ylabel("Loss")`

this code snippet should let you find an appropriate value of lr quickly.
